src/codeu/chat/common/ServerInfo.java

```
@@ -0,0 +1,36 @@
+package codeu.chat.common;
+
+/**
+ * Created by Nikki on 5/22/17.
+ */
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public final class ServerInfo {
+    private final static String SERVER_VERSION = "1.0.0";
+
+    private Uuid version;
+    public ServerInfo() {
+        try {
+            this.version = Uuid.parse(SERVER_VERSION);
+        }
+        catch(IOException e)
+        {
+            //this.version = Uuid.NULL;
```

I'm not sure you just want to silently suppress this error. If the uuid can't be parsed, that's a serious enough error to let the program fail I think.

---

The code doesn't compile unless the exception is handled.

---

I see. Then you could log an error and set this.version to Uuid.NULL.